### PR TITLE
Fixed loop global args and output, various improvements

### LINKED ISSUE #232: Global arguments and output generation are not working
"Global arguments" and "Global output" sections are empty in all loops documentation.

### PR DESCRIPTION
**Pull request by roadster31**
Fixes #232, and provides various minors improvements.

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-highlighter: rouge
```
Comment (bibich) [inline _config.yml]: According to this post : https://github.com/blog/2100-github-pages-now-faster-and-simpler-with-jekyll-3-0 Rouge is now the only supported highlighter.

Comment (roadster31) [inline _config.yml]: According to this one https://jekyllrb.com/docs/upgrading/2-to-3/#syntax-highlighter-changed, pygments is still an option.

Comment (bibich) [inline _config.yml]: Ok, but why not using the new default highlighter ? 
They are not fully compatible ?

Comment (roadster31) [inline _config.yml]: The code highlighted by rouge is hardly readable.

Anyway, I switched back to rouge. Something else will hack the css to improve readability.

**Comment by bibich** [comment]
Thanks

**Pull request merged**